pie.py

```diff
@@ -579,6 +585,47 @@ def run_pie(self,args):
             r=cmd(r'python pie.py {}'.format(' '.join(args)))
         return r
 
+    def _get_sys_prefix(self):
+        if not WINDOWS:
+            return sys.prefix
+        
+        # On Windows, running via activate.bat, sys.prefix is converted to short-name format.

```

Comment:
Love the comments, hate that we have to do this :-)

Reply:
Indeed. If you moved activation to PowerShell (which brings its own challenges seeing as the closed source Win PS is being phased out in favour of the open source PS 6+), this wouldn't be an issue, but presuming you want to keep the most basic level of support (i.e. cmd.exe), I can't think of another alternative.